packages/core/admin/admin/src/utils/strings.ts

```diff
@@ -1,3 +1,7 @@
 const capitalise = (str: string): string => str.charAt(0).toUpperCase() + str.slice(1);
 
-export { capitalise };
+function getByteSize(value: string) {
+  return new globalThis.TextEncoder().encode(value).length;
```

Comment:
I think this should be:

```suggestion
  return new TextEncoder().encode(value).length <= 72;
```